### PR TITLE
[12.0] mig account fiscal position rule

### PR DESCRIPTION
**Pull request by renatonlima**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check Travis and squash together adjacent translation commits

File: account_fiscal_position_rule/__manifest__.py [hunk 1]
```diff
@@ -8,21 +8,26 @@
...
     'name': 'Account Fiscal Position Rule',
-    'version': '10.0.1.0.0',
+    'summary': 'Account Fiscal Position Rule',
     'category': 'Generic Modules/Accounting',
-    'author': "Akretion, Odoo Community Association (OCA)",
+    'version': '12.0.1.0.0',
+    'author': (
+        'Akretion',
+        'Odoo Community Association (OCA)',
+    ),
```
Comment (legalsylvain) [inline account_fiscal_position_rule/__manifest__.py:17]: what did you changed the syntax ? 
I don't see it as the normal manner to define author. 
Ref : https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L12

File: account_fiscal_position_rule/models/account_fiscal_position_rule.py [hunk 2]
```diff
@@ -17,85 +17,84 @@ class AccountFiscalPositionRule(models.Model):
 
     name = fields.Char(
         string='Name',
-        required=True
-    )
+        required=True)
+
```
Comment (legalsylvain) [inline account_fiscal_position_rule/models/account_fiscal_position_rule.py:21]: Lot of noise here, that will be reverted in 13.0 as OCA will use black.

**Review by legalsylvain — COMMENTED** [review]
some remarks. 

the rest LGTM.

**Comment by legalsylvain** [comment]
+ could you fix travis and runbot ?

**Comment by legalsylvain** [comment]
@Tonow-c2c, @leemannd could you make a functional review of this PR ? as you seems to use this module in V10. (ref : https://github.com/OCA/account-fiscal-rule/pull/82)

thanks !

File: account_fiscal_position_rule/models/account_invoice.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    #@api.onchange('partner_id', 'company_id')
+    #def _onchange_partner_id(self):
+    #    ctx = dict(self._context)
+    #    ctx.update({'use_domain': ('use_invoice', '=', True)})
+    #    self.with_context(ctx)
```
Comment (ThomasBinsfeld) [inline account_fiscal_position_rule/models/account_invoice.py:17]: Should this method be removed or restored?

**Pull request closed without merging**